src/tests/navbar/Navbar.test.tsx

```diff
@@ -10,14 +10,16 @@ describe('Navbar', () => {
     it('Should render 4 navbar items', () => {
         const { getByTestId } = render(
             <UiConfigContext.UiConfigContextProvider projectConfig={ProjectConfigs.empty} layoutIntent={null}>
-                <Navbar
-                    projectConfig={ProjectConfigs.empty}
-                    zoom={100}
-                    undoStackState={{
-                        canRedo: false,
-                        canUndo: false,
-                    }}
-                />
+                <UiThemeProvider theme="studio">
```

Comment:
For correctness, can we use `theme="platform"` here? and in any other places

Reply:
done 184a5e897f19c083522290fe55c3faeb8245a5ea